app/javascript/pages/country/widget/components/widget-chart-tooltip/widget-chart-tooltip-component.js

```diff
@@ -1,10 +1,28 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import { format } from 'd3-format';
+import { formatCurrency } from 'utils/format';
 
 import './widget-chart-tooltip-styles.scss';
 
 class WidgetChartTooltip extends PureComponent {
+  getFormatUnit = (value, unit) => {
+    let formatUnit = '';
+    switch (unit) {
+      case '%':
+      case 'net_perc':
+        formatUnit = `${format('.1f')(value)}%`;
```

Comment:
I dont like this. We are polluting our components with state specifc info just to use a helper. We can pass this info as props and process it at the invocation. Lets discuss